PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/referencebook/AzureSdkTreePanel.java

```diff
@@ -71,6 +73,7 @@ private void initEventListeners() {
         this.tree.addTreeSelectionListener(e -> {
             final DefaultMutableTreeNode node = (DefaultMutableTreeNode) this.tree.getLastSelectedPathComponent();
             if (Objects.nonNull(node) && node.isLeaf() && node.getUserObject() instanceof AzureSdkFeatureEntity) {
+                this.lastNodePath = TreeUtil.getPathFromRoot(node);
```

Comment:
when using filter and it will auto-select the first item, in this case, we shall also record the lastNodePath

Reply:
I see only one assignment for the lastNodePath

Reply:
![image](https://user-images.githubusercontent.com/17075863/112398203-9a3b5300-8d3e-11eb-8322-5b9d667d5ee5.png)